manager/pom.xml

```diff
@@ -222,6 +231,65 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>kr.motd.maven</groupId>
+                <artifactId>os-maven-plugin</artifactId>
+                <version>${os-maven-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>detect</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.xolstice.maven.plugins</groupId>
+                <artifactId>protobuf-maven-plugin</artifactId>
+                <version>${protobuf-maven-plugin.version}</version>
+                <extensions>true</extensions>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>compile</goal>
+                            <goal>test-compile</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <protoSourceRoot>${org.gingersnap.api.basedir}</protoSourceRoot>
+                    <includes>
+                        <include>config/**/*.proto</include>
+                    </includes>
+                    <protocArtifact>
+                        com.google.protobuf:protoc:21.0-rc-1:exe:${os.detected.classifier}</protocArtifact>
+                    <extraArgs>--experimental_allow_proto3_optional</extraArgs>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <inherited>false</inherited> <!-- only execute these in the parent -->
+                <executions>
+                    <execution>
+                        <id>git submodule update</id>
```

Comment:
This breaks the Dockefile as `git` is not present. I think this should be disabled by default and only be activated if a specific profile is configured.

Reply:
is it ok to disable it for native profile?

Reply:
That would work for the images, but maybe it's better for us to be more explicit. So how about keep enabled by default, but if a dedicated profile is enabled we disable this execution?

That way we can run the native profile locally with this execution, but in the image build we can activate the profile that disables it.